x.py

```diff
@@ -11,6 +11,14 @@
 dry_run = False
 
 
+def default_java_loc():
```

Comment:
Could you rename this to `default_linux_java_loc` since it is Linux specific?

Reply:
Sure.